core/internal/gethwrappers2/compile.sh

```diff
@@ -16,6 +16,7 @@ outpath="${pkgdir}/${pkgname}.go"
 abi="${pkgdir}/${basefilename}.abi"
 bin="${pkgdir}/${basefilename}.bin"
 
+solc-select use 0.7.6
 solc --version | grep 0.7.6 || ( echo "You need solc version 0.7.6" && exit 1 )
```

Comment:
Any 0.7.6 contracts are long obsolete, so we shouldn't generate them.  (I think this is also the conclusion you came to elsewhere, right?)

Reply:
I have no idea. I was just repairing this broken script that expected solc to be that version.
IMHO we should be generating everything generated that is checked in. It may be the case that some generated code should not be used anymore, but that is a separate issue. It is currently in use, and we have no means of deterministically regenerating it :shrug:

Reply:
Yeah, I understand the motivation... I  raised a PR a while back with the same change.   I asked @connorwstein to review it since he was the last one to touch this area of the code, and he pointed out that if we're going to generate them from source contracts stored in another directory, we might as well just copy the binaries from there instead of compiling them in two different places.  He convinced me that this was better since the libocr is the "source of truth" when it comes to the contract binaries we're supposed to be testing.

Afterwards, I realized that importing them from libocr is better than copying, but... we no longer have either the source or the binary for this particular one, since we don't use it anywhere but in these tests.  That's when I opened a story for removing this dir and updating the code to import the new lib.